src/Modal/stories/index.stories.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react'
+import { ComponentStory, ComponentMeta } from '@storybook/react'
+import { X, CheckCircle } from 'phosphor-react'
+import { default as Modal, ModalProps, ModalFunctions } from '../index'
+
+export default {
+  title: 'Modal',
+  component: Modal
+} as ComponentMeta<typeof Modal>
+
+const Template: ComponentStory<typeof Modal> = (args: ModalProps) => <Modal {...args} />
+
+
+const content = "This is the content of the modal. Depends on what you want to convey, it’s best to keep it short. Else it’ll be too long.";
+
+export const Basic = Template.bind({})
+
+Basic.args = {
+  modalFunc: ()=>ModalFunctions.confirm({
+    title: 'Modal Title',
+    content: content,
+    closable:true,
+    closeIcon:<X size={16}/>,
```

Comment:
The close icon is part of the component, not the story (see figma). It can be moved to `index.ts` under `originalFunctions` like this:
`const defaultCloseIcon = <X size={16}/>;`

and then you can pass it to all modal calls as a default argument:
`closeIcon = defaultCloseIcon`